hw2/04-background.js

```diff
@@ -1 +1,31 @@
+const button = document.getElementById('button');
+let intervalStatus = 0;
 // Add your code here
+
+// https://stackoverflow.com/questions/1484506/random-color-generator
+const randomColor = function generateRandomRGB() {
+  const letters = '0123456789ABCDEF';
+  let color = '#';
+  for (let i = 0; i < 6; i += 1) {
+    color += letters[Math.floor(Math.random() * 16)];
+  }
+  color += '80';
+  return color;
+};
+
+const changeColor = function changeBackgroundColorEveryInterval() {
+  document.body.style.background = randomColor();
+};
+
+button.addEventListener('click', () => {
```

Comment:
I tried hitting Start button without imputing any value and the background color starts flickering very quickly. You may want to handle the empty inputcase to change background every 3 seconds (default time) instead of this behavior.

Reply:
I thought that I handled this .. I will check again